podcastfy/core/podcast.py

```diff
@@ -141,25 +94,22 @@ def wrapper(self, *args, **kwargs):
 
 
 class Podcast:
```

Comment:
So the idea is that a lot of the "assembling" logic in podcastfy/client.py generate_podcast() and process_links() would be replace by Podcast(), is that right?